### ISSUE TITLE
Fix dataSource definition for database connections timeouts

### ISSUE THREAD

**Issue by bclozel** [resthub/resthub-spring-stack#41]
A user reports this problem: when an application has no traffic for a long period of time, database connections timeouts occur and the pool management layer returns "dead connections".

As mentionned [here](http://stackoverflow.com/questions/1152124/jdbc4-communicationsexception) and [here](http://stackoverflow.com/questions/4950396/tomcat-configuration-using-dbcp), adding properties in the dataSource definition should solve the problem:

```
<property name="validationQuery" value="SELECT 1" />
<property name="testOnBorrow" value="true" />
<property name="testWhileIdle" value="true" />
```

Should we add these properties to RESThub dataSource definition?

**Comment by bclozel** [comment]
Fix proposal in d2dfe7d4e0

**Comment by sdeleuze** [comment]
Merged into master
